examples/resolvers-planetscale/grafbase/resolvers/products/all.ts

```diff
@@ -0,0 +1,81 @@
+import { connect, cast } from "@planetscale/database";
+
+const config = {
+  host: process.env.DATABASE_HOST,
+  username: process.env.DATABASE_USERNAME,
+  password: process.env.DATABASE_PASSWORD,
+};
+
+const conn = connect(config);
+
+export const options = {
+  cast(field, value) {
+    switch (field.name) {
+      case "id": {
+        return String(value);
+      }
+      case "onSale": {
+        return Boolean(value);
+      }
+      default: {
+        return cast(field, value);
+      }
+    }
+  },
+};
+
+export default async function ProductsAll(_, args) {
+  const { first, last, before, after } = args;
+
+  // Throw GraphQLError when arg is missing when supported
+
+  try {
+    if (first !== undefined && after !== undefined) {
+      const results = await conn.execute(
+        "SELECT * FROM Products WHERE id > ? ORDER BY id ASC LIMIT ?",
```

Comment:
I would use lower case table names which I think more common.

Reply:
Fine with me, I'll make the changes.